loom/common/platform/platformNetworkTests.cpp

```diff
@@ -32,7 +32,7 @@ SEATEST_TEST(platformNetwork_googlePing)
 {
     loom_net_initialize();
 
-    loom_socketId_t s = loom_net_openTCPSocket("google.com", 80, 1);
+    loom_socketId_t s = loom_net_openTCPSocket("google.si", 80, 1);
```

Comment:
Must have forgot it there while debugging the test that was hanging. Change that back to .com :D